trulens_eval/trulens_eval/utils/pyschema.py

```diff
@@ -627,7 +627,7 @@ def load(obj, *args, **kwargs):
                 val = obj[k]
 
             if isinstance(typ, type) \
-            and issubclass(typ, WithClassInfo) \
+            and isinstance(typ, WithClassInfo) \
```

Comment:
issubclass is needed as `typ` is a type, there is some issue with the check above. Have another PR that helps.